BookingsApi/BookingsApi.UnitTests/Mappings/JusticeUserToResponseMapperTests.cs

```diff
@@ -24,8 +24,14 @@ public void Map_MapsJusticeUser_ToJusticeUserResponse()
                 CreatedBy = "created.by@email.com",
                 Telephone = "0123456789",
             };
-            justiceUser.AddRoles(new UserRole((int)UserRoleId.VhTeamLead, "Video Hearings Team Lead"));
+            UserRole[] userRoles = new UserRole[]
```

Comment:
Not the end of the world but typically when you have the type on the right hand side then var is fine
e.g. 
```suggestion
            var userRoles = new UserRole[]
```

Reply:
ok noted. Is it something I need to change or it is ok as it is?

Reply:
personally I would change it to maintain c# coding standards as much as possible.

Reply:
changed